samples/kspsample/build.gradle.kts

```diff
@@ -42,10 +43,3 @@ kotlin {
     }
 }
 
-dependencies {
-    "kspJvmTest"("com.careem.mockingbird:mockingbird-processor")
-    "kspIosSimulatorArm64Test"("com.careem.mockingbird:mockingbird-processor")
-    "kspIosX64Test"("com.careem.mockingbird:mockingbird-processor")
-    "kspIosArm64Test"("com.careem.mockingbird:mockingbird-processor")
-}
-
```

Comment:
the plugin takes care of this now and it run kip for only 1 target instead of each target as it is by default

Reply:
🎉